src/CHANGES.txt

```diff
@@ -45,6 +45,8 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Added new module SCons.Scanner.Python to allow scanning .py files.
     - Added support for explicitly passing a name when creating Value() nodes. This may be useful
       when the value can't be converted to a string or if having a name is otherwise desirable.
+    - Fixed usage of abspath and path for RootDir objects on Windows. Previously
+      env.fs.Dir("T:").abspath would return "T:\T:" and now it correctly returns "T:".
```

Comment:
`T:` is not a directory, so `Dir("T:")` probably ought to raise an exception instead....

Reply:
As per the discussion on the Discord, T: and T:\ are usually both considered directories. But regardless, both don't work with SCons because RootDir class gives the wrong value for abspath and path.